dali/operators/decoder/nvjpeg/nvjpeg_helper.h

```diff
@@ -29,12 +29,6 @@
 #include "dali/kernels/common/copy.h"
 #include "dali/operators/decoder/image/image_factory.h"
 
-#if WITH_DYNAMIC_NVJPEG_ENABLED
```

Comment:
Why changing that?

Reply:
I had some issues saying that the function was missing during linking. I ended up moving this to a single cc file